### PR TITLE
kdeconnector module: show cell network type and strength. (#2162)

### PR DESCRIPTION
**Pull request by valdur55**
Closes ( #2162 )

### CODE REVIEW

**Comment by lasers** [comment]
@florencebiree Can you test this?

**Comment by florencebiree** [comment]
@lasers: I've just tested it, it works very well, thanks!
I think there is a possible improvement: the network strength is an integer from 0 to 4, so I added in the format line `{net_strength}/4` (reading just "2" is a bit strange).
But maybe `net_strength` could be converted in percent in the py3status code, to be something more natural to read?

**Comment by florencebiree** [comment]
I just tested the last commit of @valdur55, it's perfect! Thank you a lot!

File: py3status/modules/kdeconnector.py [hunk 1]
```diff
@@ -200,6 +213,30 @@ def _get_battery(self):
...
+        Get the connection report
+        """
+        try:
+            if self._con:
+                # Possible values are 0 - 4
+                strength = self._con.cellularNetworkStrength
+                type = self._con.cellularNetworkType
+
+                con_info = {
+                    "strength": f"{int((strength / 4) * 100)}%",
```
Comment (lasers) [inline py3status/modules/kdeconnector.py:227]: Ideally, this should stay as an int and to drop `%` and having users add `%` themselves if they want it.

Comment (valdur55) [reply py3status/modules/kdeconnector.py]: Number 0 - 4 will tell nothing until you know that max value is 4. That is why I finally decided to calculate it to %.

Comment (lasers) [reply py3status/modules/kdeconnector.py]: Not what I was trying to suggest. I want to remove `%`. 

`{strength}%` is better than adding `%` to end of `strength` value.

As an example, look at  `charge` placeholder where the value remains a number and the users can customize if they want `%` or not in the `format`.

```python
format = "{name}{notif_status} {bat_status} {charge}%" # default format
format = "{name}{notif_status} {bat_status} {charge}% {strength}%"  # percent symbols
format = "{name}{notif_status} {bat_status} {charge} {strength}"  # no percent symbols
```

**Comment by ultrabug** [comment]
Thank you @valdur55 @lasers @florencebiree

**Pull request merged**